app/src/stores/relations.ts

```diff
@@ -95,14 +90,8 @@ export const useRelationsStore = defineStore({
 
 			if (!fieldInfo) return null;
 
-			const relations: Relation[] = this.getRelationsForCollection(collection).filter((relation: Relation) => {
-				return (
-					(relation.collection === collection && relation.field === field) ||
-					(relation.related_collection === collection && relation.meta?.one_field === field)
-				);
-			});
-
-			return relations.find((relation) => relation.collection === collection && relation.field === field) || null;
+			const relations = this.getRelationsForCollection(collection);
```

Comment:
```suggestion
			const relations: Relation[] = this.getRelationsForCollection(collection);
```

Super minor but there's some inconsistent type assertion with line 59 above 🙃

Reply:
As discussed the unnecessary type assertion was removed instead via fe2571494998b1d8c771fc915043d90219e65dea